CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+## 14.11.0
+
+### Minor Changes
```

Comment:
When I'm testing the changelog for our website locally, I notice `Minor Changes` and `Patch Changes` sections will increase per release.

<img width="174" alt="image" src="https://user-images.githubusercontent.com/473530/185393016-90d6eecd-08f5-4e8a-a2cd-ef968556e5a5.png">

We may need to adjust the [script in `stylelint/stylelint.io`](https://github.com/stylelint/stylelint.io/blob/4e562026b6c5aa0aa61ad0c437dba8881a936e08/scripts/generate-docs.mjs) to prevent sections from increasing.

Reply:
Working on a PR.

Reply:
<img width="878" alt="image" src="https://user-images.githubusercontent.com/8336744/186166473-d9301cd2-98cb-4855-8ae2-0aabcc68413f.png">

Do you think this is acceptable? If so, we don't need to change anything.

Reply:
It seems better if there are no extra sections.

Reply:
> It seems better if there are no extra sections.

Forget my poor English, I'm still confused about should I remove these section titles.